Goal: Do not make CUPS the default printing system

Fixes: ?

Status wrt upstream: Debian specific

Note: We need to decide whether it's still reasonable to use "printing=bsd"
      as a default. Probably provide a debconf-supported way to upgrade

Index: samba-3.0.25rc2/source/include/includes.h
===================================================================
--- samba-3.0.25rc2.orig/source/include/includes.h	2007-04-24 11:07:02.626148126 +0200
+++ samba-3.0.25rc2/source/include/includes.h	2007-04-24 11:07:46.491777495 +0200
@@ -809,10 +809,7 @@
 #endif
 
 #ifndef DEFAULT_PRINTING
-#ifdef HAVE_CUPS
-#define DEFAULT_PRINTING PRINT_CUPS
-#define PRINTCAP_NAME "cups"
-#elif defined(SYSV)
+#if defined(SYSV)
 #define DEFAULT_PRINTING PRINT_SYSV
 #define PRINTCAP_NAME "lpstat"
 #else
